configure

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+
+function checkCompiler {
+	echo -e  "#include <stdio.h>\nint main() {\nprintf(\"hello world\\\n\");\n}" > hw.c
+	$1 -Wall hw.c -o cccheck.out 2> /dev/null
+	retcomp=$?
+	rm hw.c 2> /dev/null
+	retstr=$(./cccheck.out 2> /dev/null)
+	./cccheck.out > /dev/null 2>/dev/null
+	retexec=$?
+	rm ./cccheck.out 2> /dev/null
+	if [ $retcomp -ne 0 ] || [ "$retstr" != "hello world" ] || [ $retexec -ne 0 ]; then
+		echo ">>> Compiler failed"
```

Comment:
can you write on stderr ?
also ">>> Compiler failed" seems a little light and why ">>>".
maybe you should add the compiler use
"$CC fail simple compilation test"
you can even use the glorious error_print() (which doesn't output to stderr either, but this is a bug inside this function)